easybuild/easyconfigs/r/ReFrame-HPC/ReFrame-HPC-2.18.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonPackage'
+
+req_py_majver = 3
+req_py_minver = 5
```

Comment:
Please move this down below the `checksums =` line, above the `options =` line (that's just a matter of style, order doesn't matter w.r.t. functionality), and add a comment to clarify:

```python
# ReFrame requires Python 3.5, may only be available via 'python3'